mailpoet/tasks/npm_post_install.sh

```diff
@@ -21,3 +21,5 @@ sed -i -- "s/_focusedField\.focus()/_focusedField\.trigger('focus')/g" node_modu
 sed -i -- "s/  lastKeyIndex = keyPath.length-1;/  var lastKeyIndex = keyPath.length-1;/g" node_modules/backbone.supermodel/build/backbone.supermodel.js
 sed -i -- "s/  key = keyPath\[i\];/  var key = keyPath\[i\];/g" node_modules/backbone.supermodel/build/backbone.supermodel.js
 
+# Fix @woocommerce/components having invalid CSS not parseable by SASS
+sed -i -- "s/[(]1fr[)]/1fr /g" node_modules/@woocommerce/components/build-style/style.css
```

Comment:
Any idea what the root cause of this issue is? It would be nice if we could link to a GitHub issue and say "we can remove this code when [issue here] is fixed."

I looked at the differences between the original and what we get after running this fix, and I think they're all `-ms-grid-columns` rules like this:

![image](https://user-images.githubusercontent.com/8656640/175424767-0b0e4a62-c209-4edf-9475-e74de9567dc0.png)

I spent a bit of time trying to figure out how it ended up like this to begin with but came up empty handed.

No worries if you're not sure yourself, @websupporter.

Reply:
Sorry, I do not know :/

Reply:
I'm not aware of any existing issue, the file just contains invalid CSS (those lines you've found).